github/Organization.py

```diff
@@ -1379,6 +1394,7 @@ def _initAttributes(self):
         self._type = github.GithubObject.NotSet
         self._updated_at = github.GithubObject.NotSet
         self._url = github.GithubObject.NotSet
+        self._parent_team_id = github.GithubObject.NotSet
 
     def _useAttributes(self, attributes):
```

Comment:
Attribute `parent_team_id` of `Organization` has to be set in `_useAttributes`

Reply:
Btw., I can't see in the docs that Organization has a `parent_team_id` attribute...